src/euphorie/client/tests/test_training__hide_from_training.py

```diff
@@ -78,6 +78,9 @@ def setUp(self):
         fti_module.behaviors = tuple(
             list(fti_module.behaviors) + ["euphorie.hide_from_training"]
         )
+        from plone.dexterity.schema import SCHEMA_CACHE
+
+        SCHEMA_CACHE.invalidate("euphorie.module")
```

Comment:
wow, thanks. didn't think of the schema cache!

Reply:
I wasn't even aware it existed - I did a quick step through and got lucky because I noticed it. :smile: